popup.js

```diff
@@ -1,184 +1,158 @@
+    livelink: 'https://habtamu210.github.io/portfoliopage/',
+    sourcelink: 'https://habtamu210.github.io/portfoliopage/',
+  },
+];
+const loadCards = () => {
+  for (let i = 1; i < projects.length; i += 1) {
+    const mySection = document.createElement('section');
+    mySection.classList.add('projects');
+    mySection.id = projects[i].id;
+    mySection.innerHTML = `
+    <div class = "project-data">
+      <h3 class="project-title">${projects[i].title}</h3>
+      <p class="project-description">${projects[i].description}</p>
+      <ul class="languages-1">
+          <li class="langn">html</li>
+          <li class="langn">bootstrap</li>
+          <li class="langn">Ruby</li>
+      </ul>
+      <button class="fullwidth-button" id = ${`${projects[i].id}-btn`}>See Project</button>
+    </div>
+      `;
+    const workSection = document.querySelector('.works-1');
+    workSection.appendChild(mySection);
   }
-  
-  document.addEventListener('DOMContentLoaded', () => {
-    loadCards();
-  }, false);
-  
+};
+document.addEventListener('DOMContentLoaded', () => {
+  loadCards();
+}, false);
+
 // pup up
-const additionalLanguages = ['Github', 'Ruby', 'Bootsrap'];
 const projectButtons = [];
 const buttons = document.querySelectorAll('button');
 const modal = document.getElementById('myModal');
-
 buttons.forEach((button) => {
   const [btn] = button.id.split('-');
   if (btn === 'project') projectButtons.push(button);
 });
-
 const displayModal = (id) => {
-
   const project = projects.find((p) => p.id === id);
-    
   const render = `
   <div class="modal-container">
-
-  <div class="modal-header">
-    <div class="mod-heading">
-
-      <h2>${project.title}</h2>
-      <!-- <a href="#" class="close"></a> -->
-      <span class="close">&times;</span>
+    <div class="modal-header">
+      <div class="mod-heading">
+        <h2>${project.title}</h2>
+        <!-- <a href="#" class="close"></a> -->
+        <span class="close">&times;</span>
+      </div>
+      <ul class="modal-language">
+        <li>html</li>
+        <li>Bootstrap</li>
+        <li>Ruby rails</li>
+      </ul>
     </div>
-    
-    <ul class="modal-language">
-      <li>html</li>
-      <li>Bootstrap</li>
-      <li>Ruby rails</li>
-    </ul>
-  </div>
-
-  <div class="description-part">
-
-    <div><img class="img-b" src="${project.featureImage}" alt="feature-image"></div>
-
-    <div class="info-pop">
-      <p>${project.description}</p>
-      <div class="button-modal">
-        <button class="btn-o">See Live <img class="live" src="images/Icon-see-live.svg"></button>
-        <button class="btn-oo">See Source<img class="see"  src="images/Vector.svg"></button>
+    <div class="description-part">
+      <div><img class="img-b" src="${project.featureImage}" alt="feature-image"></div>
+      <div class="info-pop">
```

Comment:
Good job so far 😊. I think while you are trying to correct some changes in your project, a bug has been introduced and this is blocking access to the modal. When I clicked `See project` on the mobile screen, it is not displaying. Kindly fix it so that your project can get approved.⚔️😊